tests/page-objects/common/common.po.js

```diff
@@ -89,7 +92,7 @@ module.exports = {
     await helper.waitElementToPresentNative(element(by.css('.sync-status .success')));
   },
 
-  checkUserSettings: () => {
+  checkUserSettings: async () => {
```

Comment:
Seems to be no `await` here. Is it needed to be async?

Reply:
Great eye! Thanks for catching this! I've added the necessary awaits.